app/models/name.rb

```diff
@@ -145,6 +145,11 @@
 #  ranks_below_species::     Ranks: below :Species.
 #  alt_ranks::               Ranks: map alternatives to our values.
 #
+#  ==== Scopes
+#  with_correct_spelling
```

Comment:
I made this a scope because `where(correct_spelling: nil)` seems awkard to me. 
**Question**: Is `with_correct_spelling` overkill?